File: src/NocWMS-Client/src/app/models/staff.ts
import { Gender } from "./gender";

export class Staff {
    constructor(staff?: Partial<Staff>) {
        if (staff) {
            this.id = staff.id;
            this.staffName = staff.staffName;
            this.gender = staff.gender;
            this.duty = staff.duty;
            this.email = staff.email;
            this.qqNumber = staff.qqNumber;
            this.createTime = staff.createTime;
        }
    }

    public id: string;
    public staffName: string;
    public gender: Gender;
    public duty: string;
    public telephone: string;
    public qqNumber: string;
    public email: string;
    public remarks: string;
    public createTime: Date;
}

File: src/NocWMS-Server/Server.ApiService/Models/Staff.cs
using System;
using System.Collections.Generic;
using KiraNet.Camellia.Infrastructure.DomainModel.Data;
using Server.ApiService.Common;

namespace Server.ApiService.Models
{
    public class Staff : IEntity
    {
        public string Id { get; set; }
        public string UserInfoId { get; set; }
        public string StaffName { get; set; }
        public Gender Gender { get; set; }
        public string Duty { get; set; }
        public string Telephone { get; set; }
        public string Email { get; set; }
        public string QQNumber { get; set; }
        public string Remarks { get; set; }
        public DateTime CreateTime { get; set; }

        public UserInfo UserInfo { get; set; }
        public List<Warehouse> Warehouses { get; set; }
        public List<OutboundReceipt> OutboundReceipts { get; set; }
        public List<InboundReceipt> InboundReceipts { get; set; }
    }
}
